### ISSUE TITLE
Line numbers in the main windows.

### ISSUE THREAD

**Issue by drasbeck** [glushchenko/fsnotes#872]
**Description**
Adding line numbers (as an optional setting) to the main windows, can be very helpful for navigating large notes.

**Comment by gingerbeardman** [comment]
If so, "Goto Line" may also be useful.

**Comment by drasbeck** [comment]
I agree completely, I was going add this to the conversation, if it was picked up. =)
